.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Prepare a Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - debian-bookworm
```

Comment:
question: we'll need to track all building branches per package here going forward?

Reply:
Yes, if there is any special branch in the repo it needs to be tracked in this file as well as `publish-unstable`.

Reason is these branches are being used to publish to `unstable` and cut a release from.